kartothek/io_components/update.py

```diff
@@ -28,6 +29,7 @@ def update_dataset_from_partitions(
     delete_scope,
     metadata,
     metadata_merger,
+    metadata_storage_format=DEFAULT_METADATA_STORAGE_FORMAT,
```

Comment:
This is an internal function, please remove the default value. Internally we should always call it with all values

Reply:
- [] remove